src/NBitcoin/Transaction.cs

```diff
@@ -1473,7 +1473,7 @@ public TxIn AddInput(TxIn @in)
             return @in;
         }
 
-        internal static readonly int WITNESS_SCALE_FACTOR = 4;
+        public static readonly int WITNESS_SCALE_FACTOR = 4;
```

Comment:
We actually need to use the scale factor from the relevant network class instead

Reply:
👍